dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/relationship/RequestParams.java

```diff
@@ -74,11 +74,13 @@ class RequestParams extends PagingAndSortingCriteriaAdapter
     @Setter
     private List<FieldPath> fields = FieldFilterParser.parse( DEFAULT_FIELDS_PARAM );
 
+    /**
+     * @deprecated use {@link #setTrackedEntity(UID)} instead
+     */
+    @OpenApi.Property( name = "tei", value = { UID.class, TrackedEntity.class } )
+    @Deprecated( since = "2.41", forRemoval = true )
```

Comment:
I made field annotations take precedence. If I see this correct you already have an annotation on the field, just that you do not set the name in that one.

Reply:
What we need is to document query parameters

1. `trackedEntity`
2. `tei` (deprecated)

separately 😄 We are only using a field for 1. but have setters for both. This is how Spring is able to support our use case.

Is there a way for us to document them separately? With or without changing our code?

Reply:
I went the straightforward way adding another field. The solution before was neat but we'll remove the deprecated field in a later release anyway 👌🏻